packages/turf-concave/package.json

```diff
@@ -24,25 +24,29 @@
   ],
   "author": "Turf Authors",
   "contributors": [
-    "Stefano Borghi <@stebogit>"
+    "Tom MacWright <@tmcw>",
+    "Lyzi Diamond <@lyzidiamond>",
+    "Denis Carriere <@DenisCarriere>",
+    "Stefano Borghi <@stebogit>",
+    "Rowan Winsemius <@rowanwins>",
+    "Daniel Pulido <@dpmcmlxxvi>"
   ],
```

Comment:
I would also include the Authors/Contributors of `geojson-dissolve`.

- Stephen Whitmore @noffle
- Gregor MacLennan @gmaclennan